Assets/Common/Scripts/DataTable/StageTable.cs

```diff
@@ -0,0 +1,133 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using UnityEngine;
+using System.Text;
+
+
+public struct StageData
+{
+    public int Index;
+    public string StageType;
+    public string MapName;
+    public List<TableBase.ObjectData> SeedList;
+    public List<TableBase.ObjectData> MonsterList;
+}
+
+public class StageTable : TableBase
+{
+    public override void SetTable(string _Key, string _Name, string _Value)
+    {
+        StageData data;
+
+        if (mDicData.ContainsKey(_Key) == false)
+        {
+            data = new StageData();
+
+            mDicData.Add(_Key, data);
+        }
+
+        data = (StageData)mDicData[_Key];
+
+
+        switch (_Name)
+        {
+            case "Index":
+                data.Index = Int32.Parse(_Value);
+                break;
+            case "StageType":
+                data.StageType = _Value;
+                break;
+            case "MapName":
+                data.MapName = _Value;
+                break;
+            case "SeedData":
+                {
+                    data.SeedList = new List<ObjectData>();
+
+                    if (_Value.Equals("NULL") == false)
+                    {
+                        data.SeedList = GetListData(_Value);
+                    }
+                }
+                break;
+            case "MonsterData":
+                {
+                    data.MonsterList = new List<ObjectData>();
+
+                    if (_Value.Equals("NULL") == false)
+                    {
+                        data.MonsterList = GetListData(_Value);
+                    }
+                }
+                break;
+
+            default: break;
+        }
+
+        mDicData[_Key] = data;
+    }
+
+
+
+
+    public List<ObjectData> GetListData(string _Value)
+    {
+        List<ObjectData> list = new List<ObjectData>();
+
+        var splitList = _Value.Split('+');
+
+        (string, int, List<Tuple<int, int>>) value = ("", -1, new List<Tuple<int, int>>());
```

Comment:
```suggestion
        (string type_, int size, List<Tuple<int, int>> pos) value = ("", -1, new List<Tuple<int, int>>());
```

튜플에 위와 같이 Item1,2... 대신에 이름 붙여줄 수 있습니다.

[관련 링크](https://learn.microsoft.com/ko-kr/dotnet/csharp/language-reference/builtin-types/value-tuples)입니다.